src/app/utils/utils.ts

```diff
@@ -1,10 +1,11 @@
 import firebase from 'firebase';
+import { DiamPiece } from '../games/diam/DiamPiece';
```

Comment:
I missed this one. Actually I just saw in the check that I don't need to mention this one, it will block the PR, so it works \o/

Reply:
yep yep toutes les errors du linter c'est même pas nécessaire de les commenter ! Fixed.